.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,13 @@
+---
+name: "\U0001F680 Feature Request"
+about: Submit a proposal/request for a new Ingite feature
+
```

Comment:
Can we also include a label like `enhancement` here?

Reply:
I think we need to approve that a proposed FR is an enhancement.